### DISCUSSION TITLE
[AnalyticalTable] line break within data cell value

### DISCUSSION THREAD

**Discussion by SvetoslavZ99** [UI5/webcomponents-react#6606] [Q&A] [answered]
Hello,

Is there a built-in support for line breaking within a specific cell or column that I am missing from the docs?

I have a use-case where I need to visualize a couple of values, each on new line in the same cell, within the same row in one of my columns.

Tried with one of the docs demos to pass to one of the data cells a value of "**foo1\nfoo2\nfoo3**" And I saw that in the generated span title, it was working, but in the value of the span, this was stripped out. Tried passing an array of string -> did not work, it concatenated them on the same row with ",". 

Thanks in advance!

![Screenshot 2024-11-08 at 16 00 51](https://github.com/user-attachments/assets/2313ebc4-6db3-4c23-a781-7fd6b3b009c2)

**Comment by Lukas742** [comment, marked as answer]
Hi @SvetoslavZ99 per default each cell should only have one line of text, as otherwise the cell content could overflow as the table only allows static row height.

If you want to implement multiline cells, it'd recommend using a custom `Cell` renderer in the columns array and set the `rowHeight` accordingly.

https://stackblitz.com/edit/ui5wcr-at-e6zi3b?file=src%2Fdata.json,src%2FApp.tsx

**Reply by SvetoslavZ99** [reply]
Great, thank you for your quick response and proposal! I will play around with that.
